<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="US3B7Z2PF6" failures="2" tests="4" name="Test" time="11.222" errors="0" timestamp="2025-07-08T23:36:51 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="4.28"/>
  <testcase classname="api.test.UserTests" name="testGetUser" time="1.548">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testGetUser(UserTests.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testGetUser -->
  <testcase classname="api.test.UserTests" name="testUpdateUser" time="2.947">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testUpdateUser(UserTests.java:68)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUser -->
  <testcase classname="api.test.UserTests" name="testDeleteUser" time="1.432"/>
</testsuite> <!-- Test -->
